docker-compose.yml

```
@@ -26,9 +27,11 @@ services:
       - -c
       - |
         /usr/sbin/sshd &&
-        drunc-process-manager --log-level debug /process-manager-kafka.json 10054
+        drunc-process-manager --log-level debug /process-manager-kafka.json 10054 &&
```

Can make `drunc-process-manager` fork to background with a single `&`, but see main comment.
```suggestion
        drunc-process-manager --log-level debug /process-manager-kafka.json 10054 &
```